### PR TITLE
Fix/area view zooming

### PR DESCRIPTION
**Pull request by mikkoisa**
In current version when an area is selected, the map zooms to address area but then zooms to fit all areas to map.
Now if user has selected an address, the map always zooms to that area only.

### CODE REVIEW

File: src/views/AreaView/AreaView.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ const AreaView = ({
   const dispatch = useDispatch();
   const location = useLocation();
   const history = useHistory();
-  const localAddressData = useSelector(state => state.districts.districtAddressData);
+  const addressDistrict = useSelector(state => getAddressDistrict(state));
```
Comment (Dremars) [inline src/views/AreaView/AreaView.js:56]: This is fine but these could be used without needing to pass state to getAddressDistrict function. `useSelector(getAddressDistrict)`

**Review by Dremars — COMMENTED** [review]
I'm slightly confused as to what this does. I'm not able to see the difference to test version. Can you describe order of actions that should show the difference?

File: src/views/AreaView/AreaView.js (hunk 1 above)
Comment (mikkoisa) [reply src/views/AreaView/AreaView.js:56]: I think getAddressDistrict needs the state passed, since it is a Reselect selector. Otherwise it will crash.

Comment (Dremars) [reply src/views/AreaView/AreaView.js:56]: It should still get state passed since the useSelector passes state to function it receives

**Review by Dremars — APPROVED** [review]

File: src/views/AreaView/AreaView.js (hunk 1 above)
Comment (mikkoisa) [reply src/views/AreaView/AreaView.js:56]: I see! I didn't know it works that way! Much better like that 👍

**Pull request merged**